script.js

```diff
@@ -1,12 +1,194 @@
-//You can edit ALL of the code here
-function setup() {
+// Loads page.
+let setup = () => {
```

Comment:
This setup method is gloriously easy to read. I wonder if some of the other methods could be equally easy? (though they are all pretty readable, nice job!)